src/commands/set_env_vars_for_maven.yml

```
@@ -23,6 +23,6 @@ steps:
       command: <<include(scripts/set_env_vars_for_maven.sh)>>
       environment:
         CLOUDSMITH_REPOSITORY: <<parameters.repository>>
-        CLOUDSMITH_REPOSITORY_UNSTABLE: <<parameters.repository_unstable>>
+        CLOUDSMITH_REPOSITORY_SNAPSHOTS: <<parameters.repository_unstable>>
```

```suggestion
        CLOUDSMITH_REPOSITORY_SNAPSHOTS: <<parameters.repository_snapshots>>
```
```suggestion
        CLOUDSMITH_REPOSITORY_SNAPSHOTS: <<parameters.repository_unstable>>
```
Will need to change the parameter name.